### PR TITLE
fix: haproxy integration metrics description and naming

### PR DESCRIPTION
**Pull request by sigilioso**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

This PR fixes some inconsistencies in metrics units for haproxy integration. It should not be merged until [newrelic/nri-haproxy#116](https://github.com/newrelic/nri-haproxy/pull/116) is released (It will be kept as 'Draft PR' until them). We plan to release it next week.

### CODE REVIEW

**Comment by bradleycamacho** [comment]
Hi @sigilioso, thanks for the PR! I'm working on a pretty substantial structural edit to the Haproxy doc which may affect rhis PR (depending on which is merged first). If this goes live next week, it should be fine. If it goes live later than next week, we may have to chat about timing.

tldr: shoot me a ping on slack a day or so before you're planning to release this - there won't be any additional work on your part, but I may need to do some small updates on my end!

**Comment by bradleycamacho** [comment]
> Hi @bradleycamacho, thanks for letting me know 🙂. We'll do our best to unblock this as soon as possible.

No rush or anything, and don't consider it a blocker!
